pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>uk.gov.justice.services</groupId>
     <artifactId>domain-test-dsl</artifactId>
-    <version>0.3.0-SNAPSHOT</version>
+    <version>0.3.1-SNAPSHOT</version>
```

Comment:
We don't increment versions for snapshots unless we do a release when we do it automatically, keep as 0.3.0